gtsfm/two_view_estimator.py

```diff
@@ -92,7 +140,7 @@ def create_computation_graph(
 
         # verification on putative correspondences to obtain relative pose
         # and verified correspondences
-        (i2Ri1_graph, i2Ui1_graph, v_corr_idxs_graph,) = self._verifier.create_computation_graph(
+        (i2Ri1_graph, i2Ui1_graph, v_corr_idxs_graph, inlier_ratio_est_model) = self._verifier.create_computation_graph(
```

Comment:
Rename to inlier_ratio?

Reply:
Based on our other conversation on Slack, can I resolve this comment @akshay-krishnan, and leave as-is?

Reply:
Resolving